tasks/tasks.go

```diff
@@ -98,12 +98,17 @@ func (c *Congratulator) Do(ctx context.Context, repo *maintner.GitHubRepo) error
 			c.knownContributors[username] = true
 			continue
 		}
+		hasNewContributorLabel := false
 		for _, label := range ghIssue.Labels {
 			if label.Name == "new-contributor" {
-				c.knownContributors[username] = true
-				continue
+				hasNewContributorLabel = true
+				break
 			}
 		}
+		if hasNewContributorLabel {
+			c.knownContributors[username] = true
```

Comment:
I am wondering how does multiple to the single element (by `username`) to the map will create multiple results?

Reply:
see here, we wanted to `continue` over the issue (PR) that has such a label, what we did is skip over the label because the scope of that continue was for the for loop over the labels, not the issue list 🤔 https://github.com/sourcegraph/maintainerbot/pull/1/files#diff-22648141c9191e6d60ff9f6fc9a2c573L104

Reply:
Ah, gotcha, thanks!